### PR TITLE
feat: floating data table header

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
     "unplugin-auto-import": "0.16.4",
     "unplugin-vue-components": "0.25.0",
     "unplugin-vue-define-options": "1.3.7",
+    "vh-sticky-table-header": "1.5.0",
```
Comment (kelsos) [inline frontend/app/package.json:115]: How much work is to implement this functionality as part of the application?

Comment (kelsos) [reply frontend/app/package.json:115]: In general, we avoid dependencies such as this, it is a library that is small, doesn't have traction and a single person maintains it.

We will eventually run into similar issues as with the `vue-cli-electron-builder` thing. I would prefer if we don't add this kind of dependency.

Comment (kelsos) [reply frontend/app/package.json:115]: If this is a leftover from trial an error then just remove it 😄

File: frontend/app/src/types/frontend-settings.ts [hunk 2]
```diff
@@ -171,7 +171,7 @@ export const FrontendSettings = z.object({
     RefreshPeriod.default(-1)
   ),
   explorers: ExplorersSettings.default({}),
-  itemsPerPage: z.number().positive().int().default(10),
+  itemsPerPage: z.number().positive().int().default(100),
```
Comment (kelsos) [inline frontend/app/src/types/frontend-settings.ts:174]: is that change only for testing?

File: frontend/app/package.json (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/package.json]: was playing around, I need to modify a lot of the algorithms😛

**Comment by lukicenturi** [comment]
not sure why, but force push doesn't update the UI

**Pull request closed without merging**